tripy/tripy/frontend/trace/ops/slice.py

```diff
@@ -164,7 +164,9 @@ def __getitem__(
         assert np.array_equal(cp.from_dlpack(output).get(), np.arange(10)[8:2:-1])
 
     """
+    from tripy.frontend.dimension_size import DimensionSize
     from tripy.frontend.tensor import Tensor
+    from tripy.frontend.trace.ops.binary_elementwise import maximum, minimum as tensor_min
```

Comment:
Why the asymmetry here? Can we just import it as `minimum`?

Reply:
The ordinary Python `min` is used elsewhere and I didn't want to introduce confusion.

Reply:
But it's `minimum` and not `min` right?

Reply:
Yeah, just seemed like potential for confusion to me. I'll get rid of it if you don't think it's necessary.